test/acceptance_with_go_client/iterator_test.go

```
@@ -0,0 +1,90 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2025 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
```

Maybe we need to cover the following cases when testing this:

* Unset vs empty: query once with no after, once with after="", expect identical behavior (or whatever we expect for BWC).

* Invalid UUID: after="not-a-uuid" should return a clear error (do we panic now?).

* after + offset: using both together should be rejected (or precedenceused).

after + sort: not sure what happens in this case.